src/PrestaShopBundle/Command/ThemeCommand.php

```diff
@@ -0,0 +1,168 @@
+            ->addArgument('shop', InputArgument::OPTIONAL, 'Shop id on which the action will be executed');
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function init(InputInterface $input, OutputInterface $output)
+    {
+        require $this->getContainer()->get('kernel')->getRootDir() . '/../config/config.inc.php';
+        // the user must be allowed to enable themes
+        Context::getContext()->employee = new Employee(1);
+
+        $shopId = $input->getArgument('shop');
+        if ($shopId !== null) {
+            if (Validate::isLoadedObject($shop = new Shop($shopId))) {
+                Context::getContext()->shop = $shop;
+            } else {
+                $this->io->error(sprintf('The selected shop id "%s" is invalid', $shop));
+
+                return self::RETURN_CODE_FAILED;
+            }
+        }
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $this->io = new SymfonyStyle($input, $output);
+        $action = $input->getArgument('action');
+        $theme = $input->getArgument('theme');
+        $this->init($input, $output);
+
+        if (!in_array($action, $this->allowedActions)) {
+            $this->io->error(sprintf('Unknown theme action. It must be one of these values: "%s"', implode(' / ', $this->allowedActions)));
+
+            return self::RETURN_CODE_FAILED;
+        }
+
+        switch ($action) {
+            case 'export':
+                $this->executeExportThemeAction($theme);
+                break;
+            case 'install':
+                if (!filter_var($theme, FILTER_VALIDATE_URL)
+                    && !file_exists($theme)) {
+                    $this->io->error('Zip file doesn\'t exist or url is not valid');
+
+                    return self::RETURN_CODE_FAILED;
+                }
+                $this->executeGenericThemeAction($action, $theme);
+                break;
+            default:
+                $this->executeGenericThemeAction($action, $theme);
+                break;
+        }
+    }
+
+    protected function executeExportThemeAction($themeName)
+    {
+        $theme = $this->getContainer()
+            ->get('prestashop.core.addon.theme.repository')
+            ->getInstanceByName(
+                $themeName
+            )
+        ;
+
+        $path = $this->getContainer()
+            ->get('prestashop.core.addon.theme.exporter')
+            ->export($theme)
+        ;
+
+        if (false === $path) {
+            $this->io->error(sprintf('Error occured during theme export'));
+
+            return self::RETURN_CODE_FAILED;
+        }
+        $this->io->success(sprintf('Your theme has been correctly exported: "%s".', $path));
+    }
+
+    protected function executeGenericThemeAction($action, $argument)
+    {
+        /**
+         * @var \PrestaShop\PrestaShop\Core\Addon\Theme\ThemeManager
```

Comment:
namespace instead

Reply:
hello, i don't understand what actually i need to change here? the documentation? Could you explain please?

Reply:
at the top of the file
```php
use PrestaShop\PrestaShop\Core\Addon\Theme\ThemeManager;
```

and 
```suggestion
         * @var ThemeManager
```

Reply:
thanks, it's fixed